README.md

```diff
@@ -136,267 +132,43 @@ For an even more in-depth demo, check out our [Deploying to Kubernetes](https://
-```
-$ # Start Secretless Broker in another terminal on the same machine
-
-$ curl -i localhost:5335/ready
-HTTP/1.1 200 OK
-Content-Type: application/json; charset=utf-8
-Date: Thu, 27 Dec 2018 17:12:07 GMT
-Content-Length: 3
-
-{}
-```
-
-If you would like to retrieve the full informational JSON that includes details on
-which checks failed and with what error, you can add the `?full=1` query parameter
-to the end of either of the available endpoints:
-```
-$ # Start Secretless Broker in another terminal on the same machine
-
-$ curl -i localhost:5335/ready?full=1
-HTTP/1.1 200 OK
-Content-Type: application/json; charset=utf-8
-Date: Thu, 27 Dec 2018 17:13:22 GMT
-Content-Length: 45
-
-{
-    "listening": "OK",
-    "ready": "OK"
-}
-```
+## About our releases
+
+### Docker images
+The primary source of Secretless Broker releases is our [DockerHub](https://hub.docker.com/r/cyberark/secretless-broker/tags/).
+
+Every new GitHub tag (`x.y.z`) added to the project produces a set of Docker images in DockerHub: (`x.y.z`, `x.y`, `x`, `latest`). The `latest` image tag therefore always corresponds to the latest GitHub tag.
+
+### GitHub releases
+Post-1.0, GitHub releases are created for GitHub tagged versions that have undergone additional quality activities to ensure that Secretless continues to meet its [stability requirements](#stable-release-definition). When a GitHub release is created for a stable Secretless version, an additional Docker image is pushed to DockerHub with the `stable` tag.
+
+### GitHub repository
+The code on `master` in the project's GitHub repository represents the development work done since the previous GitHub tag. It is possible to build Secretless from source (see [our contributing guidelines](#development) for more info), but for regular use we recommend using the `stable` Docker image from DockerHub or an official GitHub release.
+
+### Stable release definition
+Stable components (eg service connectors, credential providers, etc.) of Secretless meet the core acceptance criteria:
+- Synchronous user actions must complete in a reasonable amount of time
+- The component must guard against threats from all parts of the [STRIDE threat model](https://en.wikipedia.org/wiki/STRIDE_(security))
+- Documentation exists that clearly explains how to set up and use the component as well as providing troubleshooting information for anticipated common failure cases.
+- A suite of automated tests that exercise the component exists, and code coverage has been measured. The entire team is in mutual agreement that any code not tested by the unit test suite is acceptable to leave untested. Tests that are deferred to be implemented at a later time are tracked so that we don't lose visibility into that work.
+  
+In addition, the following must be true for any stable release:
+- Secretless has had security review, and has no high or critical bugs
+  - Secretless has been scanned for CVEs and security vulnerabilities with no high issues
```

Comment:
what if we logged all low / medium issues in the backlog? then there would be visibility to see what remained unaddressed.

Reply:
I think that would help, particularly if we logged each individually. We can close issues we don't intend to fix and leave the pending ones open. Then, we have the path of users being able to comment on a particular issue if it is blocking them somehow or they otherwise disagree. 

I'd like to see us get to the point where even these issues don't stay on the backlog for long, but this is a good way to be transparent about what still remains.